src/controllers/HumanitecSidebarController.ts

```diff
@@ -144,6 +144,7 @@ export class HumanitecSidebarController {
                 item.id
               );
             }
+            vscode.commands.executeCommand('humanitec.score.validate');
```

Comment:
I wonder if a better solution wouldn't be to listen for configuration changes in the score validation controller instead of directly calling between them.

This way any config change would be detected, no matter how its being written.

Reply:
Good idea. I'll refactor the configuration repository with observer pattern in the next PR